google/cloud/storage/bucket_metadata.h

```diff
@@ -34,11 +91,84 @@ inline namespace STORAGE_CLIENT_NS {
  */
 class BucketMetadata : private internal::CommonMetadata<BucketMetadata> {
  public:
-  BucketMetadata() : project_number_(0) {}
+  BucketMetadata() : billing_(BucketBilling{false}), project_number_(0) {}
```

Comment:
What do you think about giving the `BucketBilling` struct a default constructor to take care of this `false` initialization?  That also allows us to more easily extend that struct if any fields get added to the bucketBilling message in the future (e.g. we don't also have to go change all occurrences of `BucketBilling{false}` to `BucketBilling{false, other_default_value}`).

Reply:
Done.